### PR TITLE
changed padView class component  to functional component

### PR DESCRIPTION
**Pull request by varunhhhrahul**
Changed PadView component in ListItem.tsx from class based to function based component, using necessary hooks and types.

### CODE REVIEW

File: src/list/ListItem.tsx [hunk 1]
```diff
@@ -109,37 +109,69 @@ type PadViewProps = {
...
-class PadView extends React.Component<PadViewProps> {
-  _root!: React.RefObject<PadView>;
+const PadView: React.FC<PadViewProps> = ({
+  children,
+  pad,
+  Component,
+  ...props
+}) => {
+  //  _root!: React.RefObject<PadView>;
+  // @ts-ignore
```
Comment (flyingcircle) [inline src/list/ListItem.tsx:119]: Not going to merge this with a `@ts-ignore` tag.

File: src/list/ListItem.tsx [hunk 2]
```diff
@@ -109,37 +109,69 @@ type PadViewProps = {
...
-            child && [
-              child,
-              index !== length - 1 && <View style={{ paddingLeft: pad }} />,
-            ]
-        )}
-      </Container>
-    );
-  }
-}
+// class PadView extends React.Component<PadViewProps> {
```
Comment (flyingcircle) [inline src/list/ListItem.tsx:145]: Why would you comment this out? If you're replacing it, remove the code

**Review by flyingcircle — CHANGES_REQUESTED** [review]

**Comment by flyingcircle** [comment]
You're failing the Travis CI build. Please review that and fix the issues.

**Review by khushal87 — COMMENTED** [review]
Looks good to me after testing 💯

**Pull request merged**